### PR TITLE
Docs/disambiguating mpi comm

### PR DESCRIPTION
**Pull request by jmlarson1**
Trying to reserve the use of `comm` for a `comms` instance and use `mpi_comm` when referring to an mpi communicator.

### CODE REVIEW

File: libensemble/libE.py [hunk 1]
```diff
@@ -270,8 +270,8 @@ def libE_mpi_worker(libE_comm, sim_specs, gen_specs, libE_specs):
     "Worker routine run at ranks > 0."
 
     from libensemble.comms.mpi import MainMPIComm
-    comm = MainMPIComm(libE_comm)
-    worker_main(comm, sim_specs, gen_specs, libE_specs, log_comm=True)
+    mpi_comm = MainMPIComm(libE_comm)
```
Comment (shuds13) [inline libensemble/libE.py:273]: This functions return is not mpi_comm. It is the libE comm object

Comment (shuds13) [reply libensemble/libE.py:273]: libE.py lines

    def __init__(self, comm):
        self.parent_comm = comm

could change the comm to mpi_comm:

    def __init__(self, mpi_comm):
        self.parent_comm = mpi_comm

Comment (shuds13) [reply libensemble/libE.py:273]: and maybe  comms_abort(comm):

**Review by shuds13 — COMMENTED** [review]
Once my comments are addressed I think this is ready to pull in.

File: libensemble/libE.py (hunk 1 above)
Comment (jmlarson1) [reply libensemble/libE.py]: All of these were addressed in 7ecae7e8

**Pull request merged**